examples/conflicts/server.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express')
+const { makeExecutableSchema } = require('graphql-tools')
+
+const { ApolloVoyagerServer, gql } = require('../../packages/apollo-voyager-server')
+
+const { conflictHandler, handleConflictOnClient } = require('../../packages/apollo-voyager-conflicts')
+
+
+// Types
+const typeDefs = gql`
+  type Hello {
+    to: String
+    ## Can be used to track conflicts
+    version: Int
+  }
+
+  type Query {
+    hello: String
+  }
+
+  type Mutation {
+    changeHello(to: String!, version: Int!): Hello
+  }
+`
+// In Memory Data Source
+let hello = {
+  to: "Stewo Hapala",
+  version: 1
+}
+
+// Resolver functions. This is our business logic
+const resolvers = {
+  Mutation: {
+    changeHello: (obj, clientData) => {
+      if (conflictHandler.hasConflict(hello, clientData)) {
+        return handleConflictOnClient(hello, clientData)
+      }
+      return hello = conflictHandler.nextState(clientData)
+    }
+  },
+  Query: {
+    hello: () => {
+      return hello.to;
+    }
+  }
+}
+
+const schema = makeExecutableSchema({ typeDefs, resolvers })
+
+// The context is a function or object that can add some extra data
+// That will be available via the `context` argument the resolver functions
+const context = ({ req }) => {
+  return {
+    serverName: 'Voyager Server'
+  }
+}
+
+// Initialize the apollo voyager server with our schema and context
+const server = ApolloVoyagerServer({
+  playground: {
+    tabs: [{
+      query: `
```

Comment:
The to make this work, each object element you provide in the `tabs` array must look something like this:

```
{
  endpoint: '/graphql',
  variables: {...},
  query: `...`
}
```

If you don't provide those properties it doesn't work.

Also, watch the formatting of the query string otherwise all of the indents show up in the playground and it looks really weird. 

![screenshot 2019-01-03 at 12 07 15](https://user-images.githubusercontent.com/8656188/50637006-255a8380-0f50-11e9-8363-6adee5ff157d.png)


Try this instead.

```js
const playgroundQuery = `mutation changeHello {
  changeHello(to: "Me!", version: 1){
    to
    version
  }
}

query hello {
  hello
}
`

// Initialize the apollo voyager server with our schema and context
const server = ApolloVoyagerServer({
  playground: {
    tabs: [
      {
        variables: {},
        endpoint: '/graphql',
        query: playgroundQuery
    }
  ]
  },
  schema,
  context
})
```